client/app/bundles/shared/components/SideNav.jsx

```diff
@@ -0,0 +1,87 @@
+// @flow
+import React from 'react';
+import { BrowserRouter as Router, Route, NavLink } from 'react-router-dom';
+import css from './SideNav.scss';
+
+const routes = [
+    {
+        path: "/dashboard",
+        exact: true,
+    },
+    {
+        path: "/moments",
+        exact: true,
+        routes: [
+            {
+            path: '/categories'
+            },
+            {
+                path: '/moods'
+            }
+        ]
+    },
+    {
+        path: "/strategies",
+        exact: true,
+    },
+    {
+        path: "/medication",
+        exact: true,
+    },
+    {
+        path: "/groups",
+        exact: true,
+    },
+    {
+        path: "/allies",
+        exact: true,
+    }
+];
+const navColor = `${css.bg}`;
+const navTwoColor = `${css.secondarybg}`;
+
+
+export default class SideNavBar extends React.Component{
+    constructor(props){
+            super(props);
+            this.state = { showMenu: false};
+        
+        this.handleClick = this.handleClick.bind(this);
```

Comment:
Need to annotate bound functions in constructor for flowtype. See https://github.com/facebook/flow/issues/5874